diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/DiktatRuleSetProvider.kt

```diff
@@ -39,6 +40,7 @@ class DiktatRuleSetProvider(private val jsonRulesConfig: String = "rules-config.
             IdentifierNaming(),
             BracesInConditionalsAndLoopsRule(),
             BlockStructureBraces(),
+            IndentationRule(),
```

Comment:
we have discussed it, but I have forgotten - as I can remember we decided to put this rule to the end of the list, no?

Reply:
Right, I will move it.